concepts/structs/.meta/config.json

```diff
@@ -1,7 +1,5 @@
 {
-    "blurb": "<todo>",
-    "authors": [
-        "misicnenad"
-    ],
-    "contributors": []
+  "blurb": "A struct is a custom data type that lets you package and name multiple related values that make up a meaningful group",
+  "authors": ["okhai"],
```

Comment:
You should use your full GH username here, see https://exercism.org/docs/building/tracks/concepts#h-file-meta-config-json